### DISCUSSION TITLE
Neuracache format still working?

### DISCUSSION THREAD

**Discussion by tedblizzard100** [ObsidianToAnki/Obsidian_to_Anki#528] [Q&A]
Hi, Is Neuracache still working?  I use the format below but it does not generate any cards.  Is there something in the set up I have to do to enable the "#flashcard" format? 

Thank you for any help.
T

Q: How do you use this style?
A: Just like this.
In Neuracache style, to make a flashcard you do #flashcard
The next lines then become the back of the flashcard

**Comment by ShootingKing-AM** [comment]
yup the format is still working !
